docker/Dockerfile.nvidia

```diff
@@ -117,6 +117,35 @@ ENV DEVITO_ARCH="nvc"
 ENV DEVITO_PLATFORM="nvidiaX"
 ENV DEVITO_LANGUAGE="openacc"
 
+########################################################################
+# NVC for GPUs via CUDA config
+########################################################################
+FROM sdk-base as nvcc
+
+# Make devito env vars file and extras
+ADD docker/nvdashboard.json /app/nvdashboard.json
+
+# mpi4py
+RUN CFLAGS=-noswitcherror /venv/bin/pip install --no-cache-dir mpi4py && rm -rf ~/.cache/pip
+    # /venv/bin/jupyter lab workspaces import /app/nvdashboard.json
```

Comment:
leftover comment?

Reply:
yikes, I think so yeah, but I copy-pasted from what's in master, so now gonna fix it all over the place

Reply:
It was commented out in the old Dockerfile.nvidia too, see here for example https://github.com/devitocodes/devito/blob/v4.6.2/docker/Dockerfile.nvidia#L147